kaldb/src/main/java/com/slack/kaldb/elasticsearchApi/OpenSearchRequest.java

```diff
@@ -191,8 +192,21 @@ private static List<KaldbSearch.SearchRequest.SearchAggregation> getRecursive(Js
                                   KaldbSearch.SearchRequest.SearchAggregation.ValueSourceAggregation
                                       .newBuilder()
                                       .setField(getFieldName(min))
+                                      .setScript(SearchResultUtils.toValueProto(getScript(min)))
```

Comment:
I missed a few spots setting the script on the min aggregator, so those are included in this PR as well.